Sources/NIOKit/NIOKitTestCase.swift

```diff
@@ -0,0 +1,35 @@
+import NIO
+import XCTest
+
+protocol NIOKitTestCaseProtocol: class {
```

Comment:
Looks like #9 was accidentally included here.

Reply:
It sure was 😳 I'd prefer to keep it in here though, and update only if #9 changes/doesn't get merged because this approach is a lot cleaner than what we had to do previously

Reply:
#9 is merged